tests/integration_tests/albumentations_test.py

```diff
@@ -0,0 +1,70 @@
+import unittest
+
+import numpy as np
+
+from super_gradients.training.datasets import Cifar10, Cifar100, ImageNetDataset
+
+try:
+    from albumentations import Compose, HorizontalFlip, InvertImg
+
+    _imported_albumentations_failure = None
+except (ImportError, NameError, ModuleNotFoundError) as import_err:
+    _imported_albumentations_failure = import_err
```

Comment:
What is the purpose of this try/catch and why it is thrown in _apply_aug down the road?

Reply:
fixed in https://github.com/Deci-AI/super-gradients/pull/495/commits/cd65908b3f3e57eebad13b15176eaa80ede1fe50